corehq/apps/users/models_sql.py

```diff
@@ -99,11 +99,11 @@ def create(cls, domain, name, permissions=None, assignable_by=None, **kwargs):
             if permissions is None:
                 # match couch functionality and set default permissions
                 permissions = Permissions()
-            role.set_permissions(permissions.to_list())
+            role.set_permissions(permissions.to_list(), sync_to_couch=False)
```

Comment:
It looks like `set_permissions` is a similar method to `set_assignable_by` where the same pattern is being implemented. Is that right? Did that have the same issue as `set_assignable_by`, and now it is being fixed?

nit: if all that is true it would have been nice to split that out in a separate commit. (no change needed now)

Reply:
yes, that is true. Both methods create related models that were not synced to couch during the method call. I will remember to try split changes up in future.